src/core/server/config/raw_config_service.ts

```diff
@@ -18,22 +18,16 @@
  */
 
 import { isEqual, isPlainObject } from 'lodash';
+import { BehaviorSubject, from, Observable } from 'rxjs';
+import { filter, map, skipWhile } from 'rxjs/operators';
 import typeDetect from 'type-detect';
-import {
-  BehaviorSubject,
-  filter,
-  k$,
-  map,
-  Observable,
-  skipRepeats,
-} from '../../lib/kbn_observable';
 
 import { ObjectToRawConfigAdapter } from './object_to_raw_config_adapter';
 import { RawConfig } from './raw_config';
 import { getConfigFromFile } from './read_config';
 
 // Used to indicate that no config has been received yet
-const notRead = Symbol('config not yet read');
+const notRead: symbol = Symbol('config not yet read');
```

Comment:
__question:__ why do need this change? The type should be inferred.

Reply:
`unique symbol` is not assignable to `any`...